### ISSUE TITLE
How many epochs is the flow generator trained for?

### ISSUE THREAD

**Issue by edgarbc** [jbohnslav/deepethogram#62]
Hi there, 

I am currently training the flow generator after adding a few videos and downloading the pre-trained models. I am using a GTX nVidia 1080 and each iteration takes around 1.5s, the loss is around 0.023 after 7 epochs. I am wondering if the number of epochs is fixed or can be stopped when reaching certain loss?

Thanks!
